modules/fluid_properties/doc/content/source/userobjects/TabulatedBicubicFluidProperties.md

```diff
@@ -0,0 +1 @@
+TabulatedFluidProperties.md
```

Comment:
This is strange. Derived classes should have their own pages. I recommend linking to a base class page or using template file(s) to get what you want.

Reply:
ok made it own class. There s really not much that is different from the base to the derived in terms of user-docs